### PR TITLE
fix: stop resource without components will delete resource

### PR DESCRIPTION
**Pull request by alexcodelf**
**Problem:**

- stop non-components resource will delete it.
- HA deployment should update

**Solution:**

- only change status for resource without components
- update HA deployment files.

**Related Issue:**
#2155

### CODE REVIEW

File: deploy/manifests/kubernetes.yaml [hunk 1]
```diff
@@ -264,7 +267,96 @@ spec:
...
+        "app.kubernetes.io/component": "minio"
+        "app.kubernetes.io/name": "minio"
+    spec:
+      volumes:
+      - name: storage
+        persistentVolumeClaim:
+          claimName: minio
+      containers:
+      - name: minio
+        image: minio/minio:RELEASE.2024-02-26T09-33-48Z
```
Comment (gitlawr) [inline deploy/manifests/kubernetes.yaml:332]: Should we update walrus-images.txt

Comment (alexcodelf) [reply deploy/manifests/kubernetes.yaml:332]: updated walrus-image.txt

**Review by gitlawr — DISMISSED** [review]
LGTM

File: deploy/manifests/kubernetes.yaml [hunk 2]
```diff
@@ -731,6 +838,8 @@ spec:
               value: $(DB_DRIVER)://$(DB_USER):$(DB_PASSWORD)@database:5432/$(DB_NAME)?sslmode=disable
             - name: SERVER_CASDOOR_SERVER
               value: http://identity-access-manager:8000
+            - name: SERVER_S3_SOURCE_ADDRESS
+              value: s3://$(MINIO_ROOT_USER):$(MINIO_ROOT_PASSWORD)@minio:9000/$(MINIO_BUCKET)?sslmode=disable
```
Comment (shanewxy) [inline deploy/manifests/kubernetes.yaml:842]: Do we need to specify this argument in docker-compose.yaml too?

Comment (alexcodelf) [reply deploy/manifests/kubernetes.yaml:842]: updated

**Review by shanewxy — APPROVED** [review]
LGTM

**Pull request merged**